code/app/src/main/res/layout/activity_main.xml

```diff
@@ -18,4 +18,10 @@
         android:layout_height="wrap_content"
         android:text="Get Local Notification" />
 
+    <Button
+        android:id="@+id/btn_triggerIAM"
```

Comment:
I would name the id `btnTriggerIAM`, since it is accessed in Kotlin using synthetic.